app/src/main/res/drawable/ic_audio_streaming_off_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:autoMirrored="true" android:height="24dp"
```

Comment:
As I understand it, we should only flip icons when they imply directionality. Otherwise, we should keep the icon the same in RTL and LTR layouts. I don't think the speaker icon falls under the directionality case, so I suggest not auto-mirroring it.

Ditto for the 'on' version.